core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/inheritance/multipleinheritanceinterface/baseannotated/childnotannotated/TestdataBaseEntity.java

```diff
@@ -0,0 +1,20 @@
+package ai.timefold.solver.core.impl.testdata.domain.inheritance.multipleinheritanceinterface.baseannotated.childnotannotated;
```

Comment:
Example of shortening:

```suggestion
package ai.timefold.solver.core.impl.testdata.domain.inheritance.multiple.interface.baseannotated.childnot;
```